dadi/frontend/components/IconArrow/IconArrow.jsx

```diff
@@ -0,0 +1,27 @@
+import { h, Component } from 'preact'
+
+import styles from './IconArrow.css'
+
+const DIRECTIONS = [
+  'left',
+  'right',
+  'down'
+]
+
+export default class IconArrow extends Component {
+  render() {
+    let classes = [styles.icon]
+
+    if (this.props.direction && DIRECTIONS.includes(this.props.direction)) {
+      classes.push(styles[this.props.direction])
+    }
+
+    return (
+      <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 36.9 20.8" class={classes.join(' ')}>
```

Comment:
Isn't that only applicable when the component is a self-closing tag, like `<Example />`? See [here](https://github.com/airbnb/javascript/tree/master/react#parentheses), for example.